doc/source/getting_started/docker/windows_container.rst

```diff
@@ -114,7 +114,7 @@ There are two build modes:
 * **Build from available binaries**: This mode builds the Docker image using
   the binaries available in the ``ansys/pyansys-geometry-binaries`` repository.
```

Comment:
Can't the name of the repository be the link so that you can omit line 117, which isn't a complete sentence.